lte/gateway/c/core/oai/tasks/mme_app/mme_app_sgs_reset.c

```diff
@@ -93,7 +93,7 @@ status_code_e mme_app_handle_sgsap_reset_indication(
  **          Return:    RETURNok, RETURNerror                              **
  **                                                                        **
  ***************************************************************************/
-status_code_e mme_app_handle_reset_indication(
+bool mme_app_handle_reset_indication(
```

Comment:
Update documentation to reflect this is returning boolean now

Reply:
Nice catch Alex!